pkg/users/users.scss

```diff
@@ -75,6 +76,14 @@
   }
 }
 
+.current-account-badge {
+  margin-left: 1.5rem;
+}
```

Comment:
You could probably drop this and add a spacing utility class such as `pf-u-ml-lg` (which means: patternfly, utility, margin: left, large spacer) to the element in the JSX instead (using `className`).

https://www.patternfly.org/v4/utilities/spacing

You'd need to make sure the utility class CSS is loaded too. But it is already added in this CSS file, with `@import "@patternfly/patternfly/utilities/Spacing/spacing.css";` at the top, so you're fine just adding the class. :+1:

Reply:
Thanks for the explanation!